src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2475,6 +2476,27 @@ private void FileTrustWarning_Click(object sender, RoutedEventArgs e)
             dynViewModel.FileTrustViewModel.ShowWarningPopup = true;
         }
 
+        private void DynamoView_Activated(object sender, EventArgs e)
+        {
+            if (dynamoViewModel.MainGuideManager != null && dynamoViewModel.MainGuideManager.currentGuide != null)
+            {
+                dynamoViewModel.MainGuideManager.ManagePopupActivation(true);
+            }
+               
+
+            if (warningPopup != null)
```

Comment:
One side note, we may want to name `warningPopup` more specific to `FileTrustWarningPopup`

Reply:
@QilongTang 
changed to fileTrustWarningPopup in the next commit: https://github.com/DynamoDS/Dynamo/pull/13002/commits/8ad9dc264e749d3de671d7a352b211ca9c3c25e7